### DISCUSSION TITLE
Kibrit çöplü oyun

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#70] [Zeka Sorusu]
Sezil ve Yeliz 60 kibrit çöpü ile bir oyun oynar. Sırası kendinde olan 1 den 6 ya kadar istediği sayıda kibrit çöpü alıyor, son kibriti alan oyunu kazanıyor.
Oyuna Sezil başlıyor, hangisi kazanır?

**Comment by Eceg25** [comment]
Çektikleri kibrit çöpü sayısına bağlı olarak değişir bu nedenle her ikisi de kazanabilir.
